entries/c/changelly.com.json

```
@@ -2,9 +2,11 @@
   "Changelly": {
     "domain": "changelly.com",
     "tfa": [
-      "totp"
+      "totp",
+      "u2f"
     ],
-    "documentation": "https://changelly.com/faq#what-is-the-2-factor-authentication?",
+    "documentation": "https://support.changelly.com/en/support/solutions/articles/14000111542",
```

```suggestion
    "documentation": "https://support.changelly.com/en/support/solutions/articles/14000111533",
```

This documentation page includes links to the YubiKey and TOTP documentation.